### PR TITLE
feat(sponsors/models.py): Migrate the images of sponsors' logo to GCS

### PR DESCRIPTION
**Pull request by hsuanchi**
## Types of changes
- [ ] **Bugfix**
- [ ] **New feature**
- [x] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
The images of the sponsors' logos are previously uploaded manually to the GCE server via Django admin. However, not every sponsor team member has access. 

- images -> Store in a dedicated GCS bucket
- Django -> Takes in the GCS path of each image

-> Sponsor team only needs the GCS IAM policy to update the logos

https://trello.com/c/TKTeVtlI/17-migrate-the-images-of-sponsors-logo-to-gcs

## Steps to Test This Pull Request
1. Go to Django Admin.
2. Create Sponsor.
3. if in dev environment -> the sponsor logo image will be save in local folder.
4. if in production env ->  the sponsor logo image will be save in google cloud storage.

### CODE REVIEW

**Comment by mattwang44** [comment]
CI failed due to the credential file is not uploaded (& it shouldn't be)

File: src/sponsors/models.py [hunk 1]
```diff
@@ -29,6 +32,7 @@ class Sponsor(ConferenceRelated):
     logo_svg = models.FileField(
         verbose_name=_('logo (SVG)'),
         blank=True, upload_to=logo_upload_to,
+        storage= default_storage if settings.DEBUG else GoogleCloudStorage(),
```
Comment (mattwang44) [inline src/sponsors/models.py:35]: Adopting the `settings.DEBUG` as the switch to determine whether to use `GoogleCloudStorage` leads to the discrepancy of migration scripts in dev & production environments, which should cause pain when trying to deploy a new version.

**Comment by mattwang44** [comment]
[Migrate the images of sponsors' logo to GCS](https://trello.com/c/TKTeVtlI/17-migrate-the-images-of-sponsors-logo-to-gcs)

**Comment by kaka-lin** [comment]
Remember rebase to new master

**Comment by hsuanchi** [comment]
Done :)

**Comment by kaka-lin** [comment]
> Done :)

**Comment by mattwang44** [comment]
Why we close this PR?

**Comment by kaka-lin** [comment]
> Why we close this PR?

My mistake, I pressed the wrong button

File: src/sponsors/models.py [hunk 2]
```diff
@@ -5,6 +5,12 @@
 
 from core.models import ConferenceRelated, BigForeignKey
 
+from storages.backends.gcloud import GoogleCloudStorage
+from django.core.files.storage import default_storage
+from django.conf import settings
```
Comment (setmao) [inline src/sponsors/models.py:10]: Group these package and order them like:
```python
from storages.backends.gcloud import GoogleCloudStorage

from django.conf import settings
from django.core.files.storage import default_storage
from django.db import models
from django.utils.translation import gettext_lazy as _
from django.utils.translation import override
from django.utils.text import slugify

from core.models import ConferenceRelated, BigForeignKey
```

**Review by setmao — CHANGES_REQUESTED** [review]

**Comment by mattwang44** [comment]
@hsuanchi @setmao 
I've opened a new PR #999 for upgrading Django. @hsuanchi please help to drop the commits for upgrading Django in this PR.

**Comment by hsuanchi** [comment]
> @hsuanchi @setmao
> I've opened a new PR #999 for upgrading Django. @hsuanchi please help to drop the commits for upgrading Django in this PR.

Done :D

**Review by setmao — APPROVED** [review]
LGTM

**Pull request merged**